website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md

```diff
@@ -1,12 +1,21 @@
 ---
-title: "Metadata API Overview"
+title: "Metadata API"
 id: "metadata-overview"
 ---
 
+## About the Metadata API
 Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse. dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata; the endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`. This API is an incredibly rich resource for evaluating data health longitudinally or at a point in time.
 
-We also provide a [graphical explorer](https://metadata.cloud.getdbt.com/graphiql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query on at any point in time.
+## Prerequisites
 
 The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud Metadata API.
 
+## How to browse the API
+
+We also provide a [graphical explorer](https://metadata.cloud.getdbt.com/graphiql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query on at any point in time.
+
+## Retention limits for data
+
+You can query data from the previous 90 days using the Metadata API.
```

Comment:
I think we should write 3 months, technically we're exposing slightly more than 90 days to ensure we cover a 3 month time line. We're using moment.js to include exactly three months, so if we to use today as an example the user could query June 27th through today.

Reply:
OK rewrote that, @ashnm!